modules/relabeling-selinux-workaround.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// migration_toolkit_for_containers/troubleshooting-mtc.adoc
+// migration_toolkit_for_containers/mtc-direct-migration-requirements.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="relabeling-selinux-workaround_{context}"]
+= Applying the Skip SELinux relabel workaround with `spc_t` automatically on workloads running on {OCP}
+
+When attempting to migrate a namespace with {mtc-full} ({mtc-short}) and a substantial volume associated with it, it is possible you encounter the `rsync-server` becoming stuck without any further information to troubleshoot the issue.
+
+To resolve this issue, set the `migration_rsync_super_privileged` parameter to `true` in both the source and destination `MigClusters` using the `MigrationController` custom resource (CR).
+
+[source,yaml]
+----
+apiVersion: migration.openshift.io/v1alpha1
+kind: MigrationController
+metadata:
+  name: migration-controller
+  namespace: openshift-migration
+spec:
+  migration_rsync_super_privileged: true <1>
+  azure_resource_group: ""
+  cluster_name: host
+  mig_namespace_limit: "10"
+  mig_pod_limit: "100"
+  mig_pv_limit: "100"
+  migration_controller: true
+  migration_log_reader: true
+  migration_ui: true
+  migration_velero: true
+  olm_managed: true
+  restic_timeout: 1h
+  version: 1.8.3
+----
+
+<1> `migration_rsync_super_privileged` indicates whether to run Rsync Pods as _super privileged_ containers, `spc_t selinux context`, or not.
+
+
+[id="diagnosis-steps-selinux-workaround_{context}"]
+== Diagnosis steps
+
+You search for and find an error of `Unable to attach or mount volumes for pod...timed out waiting for the condition` in the kubelet logs from the node where the `rsync-server` for the Direct Volume Migration (DVM) runs.
+
+
+[source,yaml]
```

Comment:
Since this is log output, it probably should not use yaml syntax highlighting. Also, consider adding a title to the codeblock (e.g. `.Example kubelet log` or something similar).